### ISSUE TITLE
Support for Client-side sorting and filtering

### ISSUE THREAD

**Issue by lauer** [brolycjw/datatable#5]
When the table does not contains data from a ActiveRecord model, but just a class which return a json data array, it would be nice to have the Client-side functionality, which is default in DataTables. 

I will try to come up with a quick solution to this, because I need it now :)

It should not try to call the sorting and filtering methods, but just return the data array directly.

**Comment by bolstycjw** [comment]
The gem is written to support server-side tables for activerecord, so you can simply use the js directly if server-side datatables are not required.

**Comment by lauer** [comment]
True, but it would be nice, just to have a setting in the Datatable class.
So the mix of ActiveRecord and in my case Flexirest keep the same structure.
